docs/getting_started/dashboard/dashboard_block_reference.md

```diff
@@ -0,0 +1,170 @@
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
+
+## Quick actions block
+
+Displays most popular/used actions and shortcuts, for example, **Create content**.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Actions** - Select actions to be displayed as shortucts: Create content, Create form, Create product, Create catalog, Create company.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of quick actions and submit your changes.
+
+## Recent activity block
+
+Displays a list of recent activity of all users.
+It also includes a link to view all activities available in Admin tab.
+
+Recent activity block contains the following data: action time, User reference (avatar, first and last name) with a link to the user profile (if available),
+Activity type with the context.
+
+Available filters are: Users, Activity type, Activity area (default value = All), Number of activities (default value = 5, maximum value = 10).
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Results limit** - Set a maximum number of activity logs to be displayed.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of content items for review and submit your changes.
+
+## Recent orders block
+
+Displays a table presenting recent orders with the newest creation date/recently placed.
+
+Table contains following columns: Order ID, Company name, Customer name, Unique items, Total value, Status, Created.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+- **Statuses** - Set the statuses of orders that should be included in the list: Pending, Processing, Cancelled, Completed. Default value = All.
+- **Limit** - Set a maximum number of orders to be displayed. Default value = 10.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of orders and submit your changes.
+
+## Review queue block
+
+Displays a list of content items which user or User group can review.
+
+On the **Properties** tab, set values in the following fields:
+
+- **Name** - Enter a name for the block.
+
+On the **Design** tab, in the **View** field, select the layout to be used to present a list of content items for review and submit your changes.
+
+## Top 10 clicked items block
+
+Displays a table presenting top 10 clicked items.
+
+Table contains following columns: Item clicked (including item name), Item type, Recommended (presenting number of recommendations of selected item),
+Clicked (presenting total number of clicks on selected item).
+
+Available filters are: Item types, Customer ID, Number of displayed items (default value = 10), Time period.
```

Comment:
```suggestion

```
Please remove this sentence.
Some are described already in Properties below (Customer ID), other are not implemented in this version.